batch/data-plane/.gitignore

```diff
@@ -38,7 +38,14 @@ local.properties
 # Other Tooling #
 .classpath
 .project
-target
+**/target/classes/**
+**/target/generated-sources/**
+**/target/generate-test-sources/**
+**/target/maven-status/**
+**/target/test-classes/com/**
+**/target/surefire-reports/**
+**/target/maven-archiver/**
+!**/target/test-classes/session-records/**
```

Comment:
Why is the `session-records` folder excluded from gitignore?  I would think everything under `target` would be ignored.

Reply:
Yeah, session-records filter can be removed now actually from batch gitignore.
Previously, test-recordings were sitting there and now they have been moved, so we can remove this.
Will push this change next.